BUILD.bazel

```diff
@@ -0,0 +1,5 @@
+# Alias the target since the globals repo isn't visible to dependent workspaces.
```

Comment:
Could you move this file into a descriptive subpackage such as `config_settings`? 

Since the alias is the entry point for users here, not the `config_setting` itself, we should explain what this does and how you could use it here.